mcux/hal_nxp.cmake

```diff
@@ -233,6 +234,10 @@ if (${MCUX_DEVICE} MATCHES "LPC5")
     include(${CMAKE_CURRENT_LIST_DIR}/mcux-sdk/drivers/vref_1/driver_vref_1.cmake)
     zephyr_include_directories(${CMAKE_CURRENT_LIST_DIR}/mcux-sdk/drivers/vref_1)
   endif()
+  if (${MCUX_DEVICE} MATCHES "LPC55S*3" AND DEFINED CONFIG_DAC_MCUX_LPDAC)
```

Comment:
Couldn't this condition be joined with the prior one for the LPADC, like so:

`if (${MCUX_DEVICE} MATCHES "LPC55S*3" AND (DEFINED CONFIG_DAC_MCUX_LPDAC OR DEFINED CONFIG_ADC_MCUX_LPADC)`

Reply:
Updated.